package.json

```diff
@@ -37,9 +40,12 @@
     "shelljs": "~0.3.0"
   },
   "devDependencies": {
+    "chai": "^2.0.0",
+    "jscoverage": "^0.5.9",
```

Comment:
Cool I've never used it I'd love to hear a more thorough reason why you're a fan of jscoverage. I like Istanbul a lot and some quick looking around leads me to think that Istanbul may be a better option. @tipiirai you care to weigh in on this? 

[Istanbul](https://www.npmjs.com/package/istanbul)
* 137,139 downloads last week
* 14 releases in 2014
* [36 contributors] https://github.com/gotwarlost/istanbul/graphs/contributors
* 1 PR merged and 2 closed this month

[jscoverage](https://www.npmjs.com/package/jscoverage)
* 4,342 downloads last week
* 4 releases in 2014
* [7 contributors](https://github.com/fishbar/jscoverage/graphs/contributors)
* 0 PRs merged 1 proposed this month